lang/perl5/Portfile

```diff
@@ -51,7 +51,11 @@ foreach {perl5.v perl5.subversion perl5.revision perl5.rmd160 perl5.sha256 perl5
 
         distname            perl-${perl5.major}.${perl5.subversion}
         dist_subdir         perl${perl5.major}
-        use_bzip2           yes
+        if {${perl5.major} >= 5.22} {
+           use_xz           yes
```

Comment:
Doesn't that mean that checksums need to be fixed as well? I could also simply be too tired at the moment to understand why it didn't fail on the Travis, so I'm sure there's something I'm not quite getting :)